samples/AzureStorage/AzureStorageLibrary/Blob/AzureBlobStorage.Codeunit.al

```diff
@@ -283,6 +285,30 @@ codeunit 50171 AzureBlobStorage
         CheckResponseCode(response);
     end;
 
+    procedure DeleteBlobs(ResourcePath: Text; Prefix: Text);
```

Comment:
Please add a comment describing the method. I will try to put comments on the other public methods.

Reply:
I think you can better describe the comment in your context ;)

What I think is: 
The DeleteBlobs operation delete a list of the blobs in a specified container. The prefix parameter delete only blobs whose names begin with the specified prefix.